### PR TITLE
Fix InspectCode documentation

### PR DESCRIPTION
**Pull request by pmiossec**
Replace ';' by ',' in C# object initializer

### CODE REVIEW

File: src/Cake.Common/Tools/InspectCode/InspectCodeAliases.cs [hunk 1]
```diff
@@ -55,11 +55,11 @@ public static void InspectCode(this ICakeContext context, FilePath solution)
...
-        ///     SolutionWideAnalysis = true;
-        ///     Profile = "./MySolution.sln.DotSettings";
-        ///     MsBuildProperties = msBuildProperties;
-        ///     OutputFile = resharperReportsDirectory + File("inspectcode-output.xml");
-        ///     ThrowExceptionOnFindingViolations = true;
+        ///     SolutionWideAnalysis = true,
+        ///     Profile = "./MySolution.sln.DotSettings",
+        ///     MsBuildProperties = msBuildProperties,
+        ///     OutputFile = resharperReportsDirectory + File("inspectcode-output.xml"),
+        ///     ThrowExceptionOnFindingViolations = true,
```
Comment (devlead) [inline src/Cake.Common/Tools/InspectCode/InspectCodeAliases.cs]: Last coma can be trimmed.

**Comment by devlead** [comment]
@pmiossec LGTM 👍. Left a minor comment, adjust, squash/rebase commit and I'll merge:+1:

**Comment by pmiossec** [comment]
done

**Comment by devlead** [comment]
@pmiossec merged, thanks for contributing 👍

**Comment by pmiossec** [comment]
No problems ;)
Continue the good work!

**Pull request merged**